docs/synapse/adminguide.rstorm

```diff
@@ -0,0 +1,1041 @@
+Synapse assumes an implicit default ``deny all`` as the final rule evaluated when checking permissions.
+
+.. NOTE::
+  
+  There are a few edge cases where a specific permission assumes a **default allow** instead of a **default deny,**
+  but these are uncommon. These cases are highly specific, and usually arise in cases where a **new** permission
+  has been implemented. That is, an action that was not originally subject to a permissions check now has one
+  (usually because of a need to explicitly **deny** that action to particular users or roles).
+  
+  If a previously unchecked action were added with "default deny", it would potentially break existing Synapse
+  deployments by suddenly blocking an action that had been previously allowed (ungated). In these circumstances
+  the new permission is given a "default allow" that can then be specifically denied if necessary.
+
+
+.. admin_global_perms:
+
+Global (Cortex) Permissions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Permissions in Synapse can be assigned at the global (Cortex) level, or to a specific Auth Gate (see
+:ref:`admin_authgate_perms`). To assign permissions to an Auth Gate, you must specify its identifier (iden)
+(i.e., using the ``--gate`` option to the appropriate Storm command) when adding the associated rule to a user
+or role.
+
+If you do not specify an Auth Gate, the permissions are **global** and apply to any / all instances within the
+Cortex where a user or role has access. For example, the following Storm command:
+
+::
+  
+  auth.role.addrule all node
+
+...grants (allows) the ``node`` permission to the built-in **all** role. This allows **any** user (because all
+users are granted the **all** role by default) to perform **any** action on **any** node in **any** layer that
+is the topmost (writeable) layer in **any** view that the user can see.
+
+Specifying rules at the global (Cortex) level may be sufficient for many basic Synapse deployments.
+
+.. NOTE::
+  
+  Recall that **order matters** when adding rules; each rule is added to the end of the list of rules assigned
+  to a user or role, and rules are evaluated in order of precedence. To reorder rules, you must:
+  
+  - remove and re-add them in the new order;
+  - use :ref:`stormprims-storm-auth-user-setRules` or :ref:`stormprims-storm-auth-role-setRules` to replace the
+    rules for a user or role with a new set of rules;
+  - use the commercial Synapse UI (Optic) to reorder rules using drag-and-drop.
+
+Assign Permissions
+******************
+
+Permissions rules (allow or deny) are assigned using the :ref:`storm-auth-user-addrule` and :ref:`storm-auth-role-addrule`
+commands.
+
+**Examples:**
+
+Prevent the user "ron" from setting tag descriptions (setting the ``syn:tag:desc`` property):
+
+.. storm-cli:: auth.user.addrule ron "!node.prop.set.syn:tag:desc"
+
+.. TIP::
+  
+  Deny rules specified with Storm must be enclosed in quotes (single or double) because they begin with
+  a symbol ( ! ).
```

Comment:
should this symbol be enclosed in backticks? (just noticed that it was the previous time it was mentioned)

Reply:
```
Deny rules specified with Storm must be enclosed in quotes (single or double) because they begin with
  a exclamation mark ``!``.
```